src/diagnostics/core_diagnostics.jl

```diff
@@ -210,6 +210,22 @@ add_diagnostic_variable!(
     end,
 )
 
+###
+# Cloud fraction (3d)
+###
+add_diagnostic_variable!(
+    short_name = "cf",
```

Comment:
```suggestion
    short_name = "cl",
```
I know it's not intuitive! lol

Reply:
oh no! where can I file a complaint about it?

Reply:
me lol, or cmip panel